source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -235,13 +235,20 @@ FilterHeadersStatus Filter::decodeHeaders(RequestHeaderMap& headers, bool end_st
     decoding_state_.setCompleteBodyAvailable(true);
   }
 
-  if (!decoding_state_.sendHeaders()) {
-    ENVOY_LOG(trace, "decodeHeaders: Skipped");
-    return FilterHeadersStatus::Continue;
+  FilterHeadersStatus status = FilterHeadersStatus::Continue;
```

Comment:
Need clarity of this exception case behavior in API comments, docs update and also version history.

Reply:
I don't see any proto comment / doc updates still in this PR.

Reply:
Yea, I haven't update the code yet. I think everyone is on the same page now. 
Will update.